lighthouse-cli/bin.ts

```diff
@@ -62,7 +62,8 @@ const cliFlags = yargs
     'list-trace-categories',
     'config-path',
     'perf',
-    'port'
+    'port',
+    'timeout'
```

Comment:
yeah I started with that but it felt so clunky and didn't make a ton of sense to have any other timeout, what about adopting the `max-wait-for-load` name from within if we change?